src/libs/elektra/CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ if (BUILD_SHARED)
 
 	add_library (elektra-kdb SHARED ${KDB_FILES})
 	add_dependencies (elektra-kdb generate_version_script)
-	target_link_libraries (elektra-kdb elektra-core)
+	target_link_libraries (elektra-kdb elektra-core elektra-ease)
```

Comment:
Not sure if I'm supposed to do create a dependency of `elektra-core` on `elektra-ease`, but otherwise I would have to duplicate the array handling code in `-core` ...